apps/fail2ban.sh

```diff
@@ -20,6 +18,26 @@ debug_mode
 # Check if root
 root_check
 
+# Check if fail2ban is already installed
+print_text_in_color "$ICyan" "Checking if Fail2Ban is already installed..."
+if is_this_installed fail2ban
+then
+    msg_box "It seems like 'Fail2Ban' is already installed.\nIf you continue, Fail2Ban and all Fail2Ban-settings get deleted."
+    if [[ "no" == $(ask_yes_or_no "Do you really want to continue?") ]]
+    then
+        exit
+    fi
+    print_text_in_color "$ICyan" "Uninstalling Fail2Ban and resetting all settings..."
+    fail2ban-client unban --all
+    check_command apt purge fail2ban -y
+    rm /etc/fail2ban/filter.d/nextcloud.conf
+    rm /etc/fail2ban/jail.local
+    msg_box "Fail2Ban was successfully uninstalled and all settings were resetted."
+    exit
```

Comment:
Why does the script exit here when it could just continue with the installation?

Reply:
because as I stated above I think it makes sense, if you can uninstall that app again. What about asking the user if he wants to continue to install the app again?

Reply:
this should be done.